be/src/column/json_column.cpp

```diff
@@ -0,0 +1,55 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "column/json_column.h"
+
+#include "util/mysql_row_buffer.h"
+
+namespace starrocks::vectorized {
+
+int JsonColumn::compare_at(size_t left, size_t right, const starrocks::vectorized::Column& rhs,
+                           int nan_direction_hint) const {
+    JsonValue* x = get_object(left);
+    const JsonValue* y = rhs.get(right).get_json();
+    return x->compare(*y);
+}
+
+void JsonColumn::fnv_hash(uint32_t* hash, uint32_t from, uint32_t to) const {
+    for (uint32_t i = from; i < to; i++) {
+        JsonValue* json = get_object(i);
+        hash[i] = json->hash();
```

Comment:
The fnv hash function should consider the input hash value

Reply:
fixed.